app/assets/stylesheets/blacklight/_bookmark.scss

```diff
@@ -37,4 +37,8 @@ label.toggle-bookmark {
   &.bookmark-unchecked {
     display: none;
   }
+}
+
+.header-tools label.toggle-bookmark {
+  min-width: 2rem;
```

Comment:
What does this style do for us? What happens without it?

Reply:
The bookmark gets moved really far to the right. I am overriding another css class that I am assuming is to keep tools formatted properly https://github.com/projectblacklight/blacklight/blob/a8b5a3d3a2e8243c8a875d40528eae5c106d83bd/app/assets/stylesheets/blacklight/_bookmark.scss#L3

Reply:
Can you add a comment into the CSS file so that other people will understand the reasoning for this rule?